sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -55,33 +58,50 @@ public synchronized void addActivity(final @NotNull Activity activity) {
       return;
     }
     frameMetricsAggregator.add(activity);
+    snapshotFrameCountsAtStart(activity);
   }
 
-  @SuppressWarnings("NullAway")
-  public synchronized void setMetrics(
-      final @NotNull Activity activity, final @NotNull SentryId sentryId) {
+  private void snapshotFrameCountsAtStart(final @NotNull Activity activity) {
+    FrameCounts frameCounts = calculateCurrentFrameCounts();
+    if (frameCounts != null) {
+      frameCountAtStartSnapshots.put(activity, frameCounts);
+    }
+  }
+
+  private @Nullable FrameCounts diffFrameCountsAtEnd(final @NotNull Activity activity) {
```

Comment:
`l`: I would move this method below `setMetrics`, because it is the only function calling this method. Now when I read `setMetrics` I have to jump all the way up to here to find the method. Keeping functions calling each other close to each other in the code improves readability, IMO.

Reply:
Thanks, forgot to reorder them